src/views/view_game.py

```diff
@@ -34,6 +35,10 @@ def init_window(self):
 
         # Display on windows
         self.window.blit(self.background, (0, 0))
+
+        self.init_game_btns()
```

Comment:
I'll definitely refactor this so buttons are updated the same way as Cards and Tokens, but that's ok FTM